### PR TITLE
TL Localization - Chapter 7, Submodules (1,112 words)

### PR DESCRIPTION
**Pull request by heywens**
Hi, kindly check my contribution. Thanks!

### CODE REVIEW

File: book/07-git-tools/sections/submodules.asc [hunk 1]
```diff
@@ -384,199 +384,199 @@ Ito ay nangangahulugan na walang lokal na tinatrabahuang branch (tulad ng ``mast
...
-Switched to branch 'stable'
+Pinalitan sa branch na 'stable'
 ----
 
-Let's try it with the ``merge'' option.
-To specify it manually, we can just add the `--merge` option to our `update` call.
-Here we'll see that there was a change on the server for this submodule and it gets merged in.
+Subukan natin ito sa opsyon na ``merge''.
+Upang itakda ito nang manu-mano, maaari lamang nating idagdag ang pagpipilian na `--merge` sa ating `update` na pagtawag.
+Dito, makikita natin na may isang pagbabago sa server para sa submodule na ito at ito ay nai-merge.
```
Comment (cherryloua) [inline book/07-git-tools/sections/submodules.asc]: Please, change "manu-mano" to "mano-mano" base on Google Translate and Glosbe it was spelled like that.

File: book/07-git-tools/sections/submodules.asc [hunk 2]
```diff
@@ -384,199 +384,199 @@ Ito ay nangangahulugan na walang lokal na tinatrabahuang branch (tulad ng ``mast
...
+Nabigo ang awtomatikong pag-merge; ayusin ang mga kasalungtan at pagkatapos i-commit ang resulta.
+Hindi ma-merge ang 'c75e92a2b3855c9e5b66f915308390d9db204aca' sa path ng submodule na 'DbConnector'
 ----
 
-You can go into the submodule directory and fix the conflict just as you normally would.
+Maaari kang magpunta sa submodule na direktoryo at ayusin ang salungatan tulad ng karaniwan mong ginagawa.
 
 [[_publishing_submodules]]
-===== Publishing Submodule Changes
+===== Paglalahatla sa mga Pagbabago ng Submodule
```
Comment (cherryloua) [inline book/07-git-tools/sections/submodules.asc]: "Paglalahatla" to "Paglalathala"

File: book/07-git-tools/sections/submodules.asc [hunk 3]
```diff
@@ -384,199 +384,199 @@ Ito ay nangangahulugan na walang lokal na tinatrabahuang branch (tulad ng ``mast
...
 
-Now we have some changes in our submodule directory.
-Some of these were brought in from upstream by our updates and others were made locally and aren't available to anyone else yet as we haven't pushed them yet.
+Ngayon ay mayroon ng tayo iilang mga pagbabago sa ating submodule na direktoryo.
+Ilan sa mga ito ay dinala mula sa upstream sa pamamagitan ng ating mga update at ang iba ay ginawa sa lokal at hindi pa magagamit ng sinuman dahil hindi pa natin na-push ang mga ito.
```
Comment (cherryloua) [inline book/07-git-tools/sections/submodules.asc]: "mayroon ng tayo" to "mayroon na tayong"

File: book/07-git-tools/sections/submodules.asc [hunk 4]
```diff
@@ -384,199 +384,199 @@ Ito ay nangangahulugan na walang lokal na tinatrabahuang branch (tulad ng ``mast
...
+  > ina-update ang setup script
+  > unicode na suporta
+  > tinanggal ang hindi kinakailangang pamamaraan
+  > nagdagdag ng bagong opsyon para sa conn pooling
 ----
 
-If we commit in the main project and push it up without pushing the submodule changes up as well, other people who try to check out our changes are going to be in trouble since they will have no way to get the submodule changes that are depended on.
-Those changes will only exist on our local copy.
+Kung tayo ay nag-commit sa pangunahing proyekto at pi-nush ito, nang wala ring pag-push sa mga pagbabago ng submodule, ang ibang tao na nagsusubok na mag-check out ng mga pagbabago ay magkakaroon ng problema dahil wala silang paraan upang makuha ang mga naka-dependeng pagbabago sa submodule.
+Ang mga pagbabago iyon ay umiiral lamang sa ating lokal na kopya.
```
Comment (cherryloua) [inline book/07-git-tools/sections/submodules.asc]: "mga pagbabago iyon" to "mga pagbabagong iyon"

File: book/07-git-tools/sections/submodules.asc [hunk 5]
```diff
@@ -384,199 +384,199 @@ Ito ay nangangahulugan na walang lokal na tinatrabahuang branch (tulad ng ``mast
...
 
-In order to make sure this doesn't happen, you can ask Git to check that all your submodules have been pushed properly before pushing the main project.
-The `git push` command takes the `--recurse-submodules` argument which can be set to either ``check'' or ``on-demand''.
-The ``check'' option will make `push` simply fail if any of the committed submodule changes haven't been pushed.
+Para makasiguro na hindi na ito mangyayari, maaari mong hilingin ang Git na suriin na ang lahat ng iyong mga submodile ay na-push nang maayos bago mag-push sa pangunahing proyekto.
```
Comment (cherryloua) [inline book/07-git-tools/sections/submodules.asc]: "maaari mong hilingin ang Git" to "maaari mong hilingin sa Git" not really sure... but the later sounds better, I guess.

File: book/07-git-tools/sections/submodules.asc [hunk 6]
```diff
@@ -384,199 +384,199 @@ Ito ay nangangahulugan na walang lokal na tinatrabahuang branch (tulad ng ``mast
...
+Ang utos `git push` ay tumatanggap ng argumento na `--recurse-submodules` na kung saan maaari itakda alinman sa ``check'' o ``on-demand''.
```
Comment (cherryloua) [inline book/07-git-tools/sections/submodules.asc]: "na kung saan maaari itakda alinman sa" to "na kung saan ay maaaring itakda ang alinman sa"

File: book/07-git-tools/sections/submodules.asc [hunk 7]
```diff
@@ -384,199 +384,199 @@ Ito ay nangangahulugan na walang lokal na tinatrabahuang branch (tulad ng ``mast
...
+Ang opsyon na ``check'' ay simpleng bibigo sa `push` kung anuman sa mga na-commit na pagbabago ng submodule ay hindi na-push.
 
 [source,console]
 ----
 $ git push --recurse-submodules=check
-The following submodule paths contain changes that can
-not be found on any remote:
+Ang mga sumusunod na mga path ng submodule ay naglalaman ng mga pagbabago na maaari
```
Comment (cherryloua) [inline book/07-git-tools/sections/submodules.asc]: "na maaari" change to "na maaaring"

File: book/07-git-tools/sections/submodules.asc [hunk 8]
```diff
@@ -384,199 +384,199 @@ Ito ay nangangahulugan na walang lokal na tinatrabahuang branch (tulad ng ``mast
...
 
-to push them to a remote.
+upang ma-push ang mga ito sa isang remote.
 ----
 
-As you can see, it also gives us some helpful advice on what we might want to do next.
-The simple option is to go into each submodule and manually push to the remotes to make sure they're externally available and then try this push again.
-If you want the check behavior to happen for all pushes, you can make this behavior the default by doing `git config push.recurseSubmodules check`.
+Tulad ng iyong nakikita, nagbibigay din ito sa atin ng ilang kapaki-pakinabang na payo tungkol sa kung ano ang maaari nating gawin sa susunod.
+Ang simpleng pagpipilian ay pumunta sa bawat submodule at manu-manong i-push sa mga remote upang matiyak na magagamit ang mga ito sa labas at pagkatapos ay subukang muli ang push na ito.
```
Comment (cherryloua) [inline book/07-git-tools/sections/submodules.asc]: "manu-manong" change to "mano-manong"

**Review by cherryloua — COMMENTED** [review]
Hi @heywens 

I have a few suggestions that you might like to consider.

**Comment by heywens** [comment]
Hi @cherryloua, thank you very much for the review. Necessary changes have been applied.

Thank you.

**Pull request merged**